src/applications/ezr/config/chapters/militaryService/supportingDocuments.js

```diff
@@ -0,0 +1,44 @@
+import environment from 'platform/utilities/environment';
+import fileUploadUI from 'platform/forms-system/src/js/definitions/file';
+import SupportingDocumentsDescription from '../../../components/FormDescriptions/SupportingDocumentsDescription';
+import content from '../../../locales/en/content.json';
+import { ezrAttachmentsSchema } from '../../../definitions/attachments';
+
+/** @type {PageSchema} */
+
+export default {
+  uiSchema: {
```

Comment:
For my understanding, where are the tests for these types of changes? It looks like we load these chapters in the form below, so are they tested on the form level somewhere else I'm not seeing?